main.go

```diff
@@ -16,8 +16,8 @@ import (
 	"github.com/fabric8-services/fabric8-cluster/jsonapi"
 	"github.com/fabric8-services/fabric8-cluster/migration"
 	"github.com/fabric8-services/fabric8-cluster/sentry"
-	"github.com/fabric8-services/fabric8-cluster/token"
 	"github.com/fabric8-services/fabric8-common/log"
+	"github.com/fabric8-services/fabric8-common/token"
```

Comment:
We should also initialize the default token manager - https://github.com/fabric8-services/fabric8-auth/blob/master/main.go#L151

Reply:
Done in https://github.com/fabric8-services/fabric8-cluster/pull/14/commits/561b4c9b7a2cfabb78d0d720274b455bd0f5b30d